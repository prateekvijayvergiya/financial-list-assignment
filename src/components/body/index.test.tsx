import Component from './'
import makeSnapshot from '../../common/testing/makeSnapshot'

describe('TableBody', () => {
  it('renders correctly', () => {
    const props = {
      items: [{
        assetClass: 'Credit',
        price: 123,
        ticker: 'ALPHA'
      }]
    }
    makeSnapshot(<Component {...props}/>)
  })
})
