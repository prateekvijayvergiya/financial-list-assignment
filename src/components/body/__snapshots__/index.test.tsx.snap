// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableBody renders correctly 1`] = `
<tbody>
  <tr
    className="sc-bdnxRM hZOTJo"
  >
    <td
      className="sc-gtsrHT JPHAX"
    >
      Credit
    </td>
    <td
      className="sc-dlnjwi crbEhw"
    >
      Rs. 
      123
    </td>
    <td
      className="sc-gtsrHT JPHAX"
    >
      ALPHA
    </td>
  </tr>
</tbody>
`;
